openfga/v1/openfga_service.proto

```diff
@@ -1588,6 +1586,8 @@ message Assertion {
     json_name = "expectation",
     (google.api.field_behavior) = REQUIRED
   ];
+
+  openfga.v1.ContextualTupleKeys contextual_tuples = 3 [json_name = "contextual_tuples"];
```

Comment:
Instead of this, can we follow the example in `ListUsers`?

```
https://github.com/openfga/api/blob/7e5be7b65c273bdcbddb7b88e98a9b7495effa05/openfga/v1/openfga_service.proto#L892-L896
```